File: backend/src/main/java/org/jdk/project/dto/config/DeviceUpsertRequest.java
package org.jdk.project.dto.config;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class DeviceUpsertRequest {
  private Long loungeId;
  private String deviceName;
  private String deviceType;
  private String deepGlintDeviceId;
  private Integer enable;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/DeviceUpsertRequest.kt
package org.jdk.project.dto.config

data class DeviceUpsertRequest(
    var siteId: Long? = null,
    var deviceName: String? = null,
    var deviceType: String? = null,
    var deepGlintDeviceId: String? = null,
    var enable: Int? = null,
    var remark: String? = null,
)
